charts/nri-kube-events/README.md

```diff
@@ -1,57 +1,149 @@
+| [global.]verboseLog | `false` |  | Sets the debug logs to this integration or all integrations if it is set globally |
+
+### Further information
+<a name="values-managed-globally-1"></a>
+#### 1. Merged
+
+Merged means that the values from global are not replaced by the local ones. Think in this example:
+```yaml
+global:
+  labels:
+    global: global
+  hostNetwork: true
+  nodeSelector:
+    global: global
+
+labels:
+  local: local
+nodeSelector:
+  local: local
+hostNetwork: false
 ```
+
+This values will template `hostNetwork` to `false`, a map of labels `{ "global": "global", "local": "local" }` and a `nodeSelector` with
+`{ "local": "local" }`.
+
+This is the rationale behind of this behavior:
+ * `hostNetwork` is templated to `false` because is overriding the value defined globally.
+ * `labels` are merged because the use may want to label all the New Relic pods at once and label other solution pods differently for
+   clarity' sake.
+ * `nodeSelector` does not merge as `labels` because could make harder to overwrite/delete a selector that comes from global because
+   of the logic that Helm follows merging maps.
+
+It could be confusing to have two behaviours regarding when this chart merges or replaces the `values` from global to local. All functions
+simply replace the value from `global` with the one that is in the root of the chart except functions that template `labels`,
+`customAttributes` and service account's annotations.
+
+<a name="values-managed-globally-2"></a>
+#### 2. Fine grain registries
+
+There are charts that only have 1 image and others that can have 2 or more images. The local path for the registry can change depending
+on the chart itself. If you have an advanced configuration that has a proxy cache for the docker registry you should take a look to the
+chart's `values.yaml` directly to see where you have to change it if you really need so much granularity that needs a proxy/cache registry
+per integration.
+
+<a name="values-managed-globally-3"></a>
+#### 3. Privileged mode
+
+By default, from the common library, the privileged mode is set to false. But most of the helm charts require this to be true to fetch more
+metrics so could see a true in some charts. The consequences of the privileged mode differ from one chart to another so for the chart that
+honors the privileged mode toggle should be a section in the README explaining which is the behavior with it enabled or disabled.
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| affinity | object | `{}` |  |
+| agentHTTPTimeout | string | `"30s"` |  |
+| containerSecurityContext | string | `nil` |  |
+| customAttributes | object | `{}` | Custom attributes here has the same functionality as in the new infrastructure v3 chart. you can read a string (because is a string in JSON format) or read directly the map here and encode to JSON. |
+| deployment.annotations | object | `{}` |  |
```

Comment:
Is it normal that most of the options do not have the description?

Reply:
this is what you were mentioning in the PR description, no?

Reply:
Yep. It has been done [here](https://github.com/newrelic/nri-kube-events/pull/112/commits/a84dee1c8948a0684b00a8f8a7554fecc27803d7)